examples/configfile/builtin/main.go

```diff
@@ -28,14 +28,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	cfg "sigs.k8s.io/controller-runtime/pkg/config"
 	"sigs.k8s.io/controller-runtime/pkg/log"
-	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/log/bootstrap"
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
 var scheme = runtime.NewScheme()
 
 func init() {
-	log.SetLogger(zap.New())
+	log.SetLogger(bootstrap.New())
```

Comment:
It is a breaking change for those who are importing controller runtime. 
Can you please use the emoji `:warning:` ? 

Also, do we need to change the lib?